bittensor_cli/cli.py

```diff
@@ -2804,7 +2804,7 @@ def stake_get_children(
         wallet_path: Optional[str] = Options.wallet_path,
         network: Optional[str] = Options.network,
         chain: Optional[str] = Options.chain,
-        netuid: int = Options.netuid,
+        netuid: Optional[str] = Options.netuid,
```

Comment:
No. Args should be one thing. You can have a separate `all_netuids` arg, but let's not have this single arg doing numerous jobs. Look at the other methods that utilise `--all` flags for guidance. Other than that, looks fine.

Reply:
Changed from `btcli stake get_children --netuid all` to `btcli stake get_children --all-netuids` and `btcli stake get_children --all`